improver/__init__.py

```diff
@@ -47,55 +47,58 @@ class BasePlugin(ABC):
     Subclasses must be callable. We preserve the process
     method by redirecting to __call__.
     """
-
-    def __call__(self, *args, **kwargs):
-        """Makes subclasses callable to use process
+    def __call__(self, *args, verbose=False, **kwargs):
```

Comment:
Not strictly necessary adding this verbose argument.  Perhaps I presume too much to do this??